frontend/src/app/features/admin/components/admin-root/admin-root.component.html

```diff
@@ -1,33 +1,44 @@
-<mat-toolbar
-  color="primary"
->
-  <div class="main-menu">
-    <button
-      mat-flat-button
-      color="primary"
-      class="menu-element"
-      [routerLink]="adminRoutes.Settings"
+<div *transloco="let t; read:'admin-root'">
```

Comment:
Ide egy `ng-container` is elég lenne, mert ennek a div-nek semmi szerepe nincs a renderelt DOM-ban.